rules/python_rules.build_defs

```diff
@@ -16,7 +16,7 @@ a target which has only had small changes.
 
 def python_library(name:str, srcs:list=[], resources:list=[], deps:list=[], visibility:list=None,
                    test_only:bool&testonly=False, zip_safe:bool=True, labels:list&features&tags=[], interpreter:str=None,
-                   strip:bool=False):
+                   strip:bool=False, strip_package_name:bool=CONFIG.PYTHON_STRIP_PLEASE_PACKAGE_NAMES):
```

Comment:
Are there other use cases for this, or is it specific to trying to produce a python package you can publish to a repo e.g. pypi? 

It might be more descriptive to call it something like `PYTHON_PACKAGE_ROOT` if this is the roots of the python packages we're installing. The fact we're implementing this by stripping the paths in the pex tool is really an implementation detail IMO.

Reply:
I think something like `PYTHON_USE_PACKAGE_ROOT` would be better since it's a boolean. `PACKAGE_ROOT` sounds like an option to just package the roots.